src/main/kotlin/no/skatteetaten/aurora/boober/feature/StsFeature.kt

```diff
@@ -73,124 +65,56 @@ class StsDisabledFeature : Feature {
 }
 
 @Service
-@ConditionalOnProperty("integrations.skap.url")
-class StsFeature(val sts: StsProvisioner) : Feature {
+@ConditionalOnProperty(value = ["integrations.skap.url", "integrations.bigbird.url"])
```

Comment:
Bør vi bruke en custom @ConditionalOn i dette tilfellet? 
Å teste på property integrations.skap.url gir god mening ved oppretting av RestTemplate. Men StsFeature (og flere andre komponenter) har ikke noe direkte avhengighet til denne propertien.

Reply:
God spm. Problemet er at hvis vi ikke skrur av denne vil ikke appen starte opp siden StsProvisioner ikke vil kunne lages. Jeg er åpen for alternative forslag her.